wa/starter/src/main/resources/log4j2.xml

```diff
@@ -37,9 +37,14 @@ under the License.
 
   <loggers>
 
-    <asyncLogger name="org.apereo.cas" additivity="false" level="INFO">
+    <asyncLogger name="org.apereo.cas" additivity="false" level="DEBUG">
```

Comment:
I think this `DEBUG` and the one following should be reverted to `INFO` - this file is supposed to be used by deployments.

Reply:
No objections, but wouldn't you say that having DEBUG level on is incredibly useful for a deployment as a starting point to troubleshoot and figure out what's going on? That is [the very first thing](https://apereo.github.io/cas/development/installation/Troubleshooting-Guide.html#review-logs) we do for deployment, or ask people to do when they trying to diagnose issues.

Reply:
OTOH, I think we should revert. You're right; I would prefer the system to be production ready OOTB.